frontend/src/features/properties/list/utils.ts

```diff
@@ -0,0 +1,23 @@
+import { IPaginateProperties } from 'constants/API';
```

Comment:
Putting url generation in a utils file doesn't really match up with what we've been doing with our other api calls. Mostly we've been centralizing urls in src\hooks\pims-api\use*

Reply:
This was hardcoded at the top of the PropertyListView file. I just extracted to a separate file. I don't think a function that generates a URL warrants a custom "hook" - or does it?

Reply:
strictly speaking, no. This was a convention that Jeremy started when he refactored parts of the application. We don't have to keep using it if it isn't providing value.

At a minimum I think I'd still like to see utils renamed to something more specific. 

We have been working to centralize API calls as well, so moving this out of the feature specific area would be nice (optional).